thirdeye-ui/src/app/components/rca/top-contributors-table/top-contributors-table.component.tsx

```diff
@@ -76,6 +133,17 @@ export const TopContributorsTable: FunctionComponent<TopContributorsTableProps>
                         <TableCell>
                             <strong>{t("label.impact-percentage")}</strong>
                         </TableCell>
+                        <SortableTableHeader onClick={toggleSort}>
+                            <strong>{t("label.change-percentage")}</strong>
```

Comment:
While I understand the use case here, it is better to avoid nesting ternary operators like this. 

- Can use a value returned by `useMemo` or something similar with a list of branching if-return statements, so it is easier to understand what is happening
- Alternatively, can have multiple rendering clauses, like
```jsx
{changePercentSort === null && (
  <></> //Is this empty fragment needed for proper rendering?
)}
{changePercentSort === TopContributorsTableChangePercentSort.ASC && (
  <KeyboardArrowUp />
)}
{changePercentSort === TopContributorsTableChangePercentSort.DESC && (
  <KeyboardArrowDown />
)}
```

Not a blocker comment, can fix this later if the PR is urgent.